src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -344,18 +344,32 @@ public static string FormatInvariant(string format, params object[] args)
 
         private static string RemovePII(object arg)
         {
-            if (arg == null)
-                return string.Empty;
-
             if (arg is Exception ex && ex.GetType().FullName.StartsWith("Microsoft.IdentityModel.", StringComparison.Ordinal))
```

Comment:
can you reference IsCustomException here to decrease code duplication?

Reply:
Updated

Reply:
Updated